app/Models/Story.php

```diff
@@ -8,4 +8,9 @@
 class Story extends Model
 {
     use HasFactory;
-}
+
+    public function stories()
+    {
+        return $this->hasMany(Story::class);
```

Comment:
Questa è sbagliata: la storia appartiene ad una Epica. Quando scrivi queste funzioni devi sempre mettere dal punto di vista del modello. Il Modello in questo caso è la storia e quindi una storia appartiene ad una Epica. La funzione quindi si deve chiamare epic e non Stories, inoltre si deve chiamare la funzione belongsTo e non la hasMany

`

public function epic() {
    return $this->belongsTo(Epic::class);
}

`